docs/tutorials/goldenpath.md

```diff
@@ -0,0 +1,497 @@
+
+
+
+## Creating an object to spawn for each connected player
+
+1. Create 3D Object->Capsule (name it **Player**). 
+1. Add a `NetworkObject` component.
+1. Click the Assets folder.
+1. Create a new Folder and call it **Prefabs**.
+1. Make **Player** a prefab by dragging it to **Prefabs** folder you just created.
+1. Delete **Player** from scene.
+
+  :::tip
+  We remove **Player**, because we will be using the network library to spawn the player. The library cannot track objects that start in the scene.
+  :::
+
+1. Select `NetworkManager`.
+1. Inside the `NetworkManager` component tab, locate the  `NetworkPrefabs` field. 
+1. Click `+` to create a slot
+1. Drag this player prefab from above into the new empty slot
+
+  :::tip
+  When you select the **Default Player Prefab** , you are telling the library that when a client connect to the game, automatically spawn this prefab as the character for the connecting client. If you do not have the default selected for any prefab the game will crash on client connect.
+  :::
+
+  :::note
+   You may see the following error reported `There is no NetworkPrefab Marked as a PlayerPrefab`. Once you have completed the above steps you can clear the error.
+  :::
+
+1. Create a 3D Object->Plane, centered at (0,0,0).
+1. Save your scene
+
+<iframe src="https://www.youtube.com/embed/B_FWb4J1Pxw?playlist=B_FWb4J1Pxw&loop=1&&autoplay=0&controls=1&showinfo=0&mute=1"   width="854px"
+        height="480px" className="video-container" frameborder="0" position="relative" allow="accelerometer; autoplay; loop; playlist; clipboard-write; encrypted-media; gyroscope; picture-in-picture"  allowfullscreen=""></iframe>
+
+
+### Testing the basic network building blocks
+
+1. To do a quick test you can click play.  
+1. The Editor will start, and you will just see the plane.  
+1. Without stopping the editor's play mode, navigate to the `NetworkManager` component in the Hierarchy tab (it will be underneath `DontDestroyOnLoad`).  
+1. In the `NetworkManager` inspector scroll down and find the `Start Host` button.  
+1. If you click it, you will see the player capsule spawn. 
+1. Stop the player.
+
+<iframe src="https://www.youtube.com/embed/iLb00icvRMs?playlist=iLb00icvRMs&loop=1&&autoplay=0&controls=1&showinfo=0&mute=1"   width="854px"
+        height="480px" className="video-container" frameborder="0" position="relative" allow="accelerometer; autoplay; loop; playlist; clipboard-write; encrypted-media; gyroscope; picture-in-picture"  allowfullscreen=""></iframe>
+
+
+## Creating a command line helper
+This command line helper will help us launch our project from a command line.
+
+1. Click the Assets folder.
+1. Create a new Folder and call it **Scripts**.
+1. Create a script called `NetworkCommandLine`.
+1. Right click on the `NetworkManager` in the hierarchy view.
+1. Create an empty GameObject underneath it.
+1. Rename it `NetworkCommandLine`.
```

Comment:
```suggestion
1. Right click on the `NetworkManager` in the hierarchy view, and choose "Create Empty". This will create an empty GameObject, with `NetworkManager` as its parent.
1. Rename this child GameObject `NetworkCommandLine`.
```

Reply:
Done